Neos.Flow/Classes/Security/Policy/PolicyService.php

```diff
@@ -279,11 +292,13 @@ public function getPrivilegeTargets()
      *
      * @param string $privilegeTargetIdentifier Identifier of a privilege target
      * @return PrivilegeTarget
```

Comment:
```suggestion
     * @return ?PrivilegeTarget
```

Reply:
Do we actually use the `?` optional type prefix in docblocks? IIRC our current reflection does not parse that correctly - the alternative is `|null`

Reply:
yes, you're right. Updated the suggestion!